### PR TITLE
Resync master for 5.1

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR resyncs `master-post-release` with master to pull in changes landed for 5.1

/cc @alanconway @vimalk78

### CODE REVIEW

**Review by alanconway — COMMENTED** [review]
/lgtm

**Review by vimalk78 — CHANGES_REQUESTED** [review]
some tests are failing because there is a mismatch between the destination fluent pod name
The test starts a pod named `fluentForward` but later tests look for logs in `fluentdForward`


https://github.com/openshift/cluster-logging-operator/blob/9c0fe1ef2cf673792cc995c24d850a1c306d6b74/test/functional/cluster_log_forwarder.go#L8
Change
```
-       fluentForwardOutputName        = "fluentForward"
+       fluentForwardOutputName        = "fluentdForward"
```

https://github.com/openshift/cluster-logging-operator/blob/9c0fe1ef2cf673792cc995c24d850a1c306d6b74/test/functional/normalization/message_format_test.go#L181
Change
```
-               raw, err := framework.ReadApplicationLogsFrom("fluentforward")
+               raw, err := framework.ReadApplicationLogsFrom(logging.OutputTypeFluentdForward)
```

**Comment by vimalk78** [comment]
please /retest

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ const (
 // COMPONENT_IMAGES are thee keys are based on the "container name" + "-{image,version}"
 var COMPONENT_IMAGES = map[string]string{
 	"curator":             "CURATOR_IMAGE",
```
Comment (vimalk78) [inline pkg/utils/utils.go:34]: this was deleted in earlier commit in this PR, but added back in the subsequent commit

**Pull request merged**